add doc for missing OMV core plugins (#69)
openmediavault/openmediavault-docs: plugins.rst

The following is the list of official plugins by |omv|.

* **ClamAV**: Provides Clam AntiVirus (ClamAV). It is a free software, cross-platform and open-source antivirus software toolkit able to detect many types of malicious software, including viruses

> * **Diskstats**: Complementary plugin to extend system statistics collection by adding iostats graphs.

Reviewer: Could you please change that to the following. I think this phrasing is more clear than the original (moved) text.
```suggestion
* **Diskstats**: Complementary plugin to extend system statistics collection by adding I/O statistic graphs.
```

* **Forked-daap**: Provides a daap protocol music server.
* **LVM2**: LVM managing. Create volume groups and logical partitions.
* **NUT**: Controlling and configuring UPS. The driver support is based on NUT.

> * **ShAirPort**: Provides Airtunes emulator. Stream music wirelessly to your iPod/iPad/iPhone/iTunes.

Reviewer: I'm not sure, but i think it is named `Shairport`, at least according to the [Git repository](https://github.com/mikebrady/shairport-sync) of the project.

> * **ShareRootFs**: Provides shared directires on root file sstem

Reviewer: ```suggestion
* **ShareRootFs**: Provides shared directories on root file system.
```

Reviewer: This issue hasn't been fixed.

> * **SNMP**: Provides Simple Network Management Protocol (SNMP). SNMP is an Internet Standard protocol for collecting and organizing information about managed devices on IP networks 

Reviewer: Most enumerations end with a dot, so we should do the same here.
```suggestion
* **SNMP**: Provides Simple Network Management Protocol (SNMP). SNMP is an Internet Standard protocol for collecting and organizing information about managed devices on IP networks.
```

> * **TFtp**: Provides Trivial File Transfer Protocol (TFTP). TFTP is a simple lockstep File Transfer Protocol which allows a client to get a file from or put a file onto a remote host

Reviewer: ```suggestion
* **TFtp**: Provides Trivial File Transfer Protocol (TFTP). TFTP is a simple lockstep File Transfer Protocol which allows a client to get a file from or put a file onto a remote host.
```
